dkron/api.go

```diff
@@ -248,3 +256,13 @@ func (h *HTTPTransport) jobToggleHandler(c *gin.Context) {
 	c.Header("Location", c.Request.RequestURI)
 	renderJSON(c, http.StatusOK, job)
 }
+
+// isSlug determines whether the given string is a proper value to be used as
+// key for in any backend store (a "slug"). If false, the 2nd return value
```

Comment:
We're only using one store now in v2.

```suggestion
// key for the backend store (a "slug"). If false, the 2nd return value
```

Reply:
Done.
v2 uses only one store? Which one?

Reply:
Found it: BadgerDB.